our-work/software/cogresnet.yml

```diff
@@ -0,0 +1,20 @@
+title: Cognitive Research Registration App
+description: "Firebase Application that allows participants to register for studies, and for researchers to manage participant registrations"
+languages:
+  - javascript
+  - vuejs
+tags:
+  - cognitive-science
```

Comment:
is cognitive-science meaningfully different from neuro?

Reply:
CLPS is a different department than Neuroscience, so yes? Neuro doesn't need to study behavior in any way, it could be movement, hearing... but yes, there is overlap when cognitive-science tries to tie to the behavior of the brain.... I'm not sure that it always needs to... I partly feel that if they come from different departments we should acknoweled that. For soma we put `computational-psychiatry` ... that's not a department, but Rike uses that term always when describing what she does. The can add `neuro` as well? The faculty are mixed in the `Cognitive Research Network` come from CLPS and Neuroscience

Reply:
Ah... should borton have `cognitive-science` as well then as they do a lot of behavioral tasks?

Reply:
I can ask him. They do it for clinical reasons...`ocd`... that may be closer to the `computational psychiatry` but these boundaries are tough. I've never heard him say that his research is in `cognitive-science`. I  don't think his lab designs/optimizes tasks based on models that they are evaluating. They instead use what other researchers recommend as the proper way to measure "X". We could probably just ask the collaborators! I can ask Dave for his input and we could ask Rike at a meeting or something and see how we refine? Beehive should probably include that tag `cognitive-science` though, so I'll add that